### PR TITLE
Ft docker config 173033208

### PR DESCRIPTION
**Pull request by Musyimi97**

### CODE REVIEW

File: IdeaHub/settings.py [hunk 1]
```diff
@@ -140,11 +143,11 @@
 # Database
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
-DATABASE_NAME = os.getenv('DATABASE_NAME')
-USER_NAME = os.getenv('USER_NAME')
-DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD')
-DATABASE_HOST = os.getenv('DATABASE_HOST')
-DATABASE_PORT = os.getenv('DATABASE_PORT')
+DATABASE_NAME = os.getenv('DATABASE_NAME', os.path.join(BASE_DIR, "db.sqlite3"))
```
Comment (Ogutu-Brian) [inline IdeaHub/settings.py:146]: Hello @Musyimi97 ,
We will not be using sqlite for the database so I don't think we need to do this, do we?

File: IdeaHub/settings.py [hunk 2]
```diff
@@ -156,7 +159,7 @@
 else:
     DATABASES = {
         'default': {
-            'ENGINE': 'django.db.backends.postgresql',
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (Ogutu-Brian) [inline IdeaHub/settings.py:162]: Hi @Musyimi97 ,
Is there a reason why we need to change the database Engine here? 
Kindly have a look at [this documentation](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-DATABASE-ENGINE)
Screenshot 2020-06-01 at 21 50 15

File: README.md [hunk 3]
```diff
@@ -23,12 +23,19 @@
 # Setting up the project
 * Clone the repo : `git clone https://github.com/Ogutu-Brian/IdeaHub-API.git`
 * Navigate to the cloned folder `$ cd IdeaHub-API/`
-* Run `$ pipenv shell` . This will create a virtual environment and pipfile for package requirements.
-* Run `$ pipenv install` to install the packages in your environment.
-* $ touch .env
-* copy content of example_env.txt to .env and update the values to your own configs
-* Run migrations `python manage.py migrate`
-* Start the server `python manage.py runserver`
+* Run `$ docker-compose up -d --build` . This is to build the docker image and install all dependencies
```
Comment (Ogutu-Brian) [inline README.md:26]: Hi @Musyimi97 ,
Great job in updating the documentation.
Apart from using docker to run the app, one should be able to set it up manually without docker. So we still need to retain the original setup too.
You could add a section like:
**Set up with docker**
That being said, I don't think we need to completely do away with the original documentation. What do you think?

File: IdeaHub/settings.py (hunk 1 above)
Comment (Musyimi97) [reply IdeaHub/settings.py:146]: Getting rid of it asap

File: IdeaHub/settings.py (hunk 2 above)
Comment (Musyimi97) [reply IdeaHub/settings.py:162]: No we will be using PostgreSQL, thanks for heads up

File: README.md (hunk 3 above)
Comment (Musyimi97) [reply README.md:26]: Totally fine with that decision. lemme work on this

**Pull request closed without merging**